### PR TITLE
Msw 114008595 update learn logic

### PR DESCRIPTION
**Pull request by michaelwehrley**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -329,6 +329,3 @@ DEPENDENCIES
   think_feel_do_engine!
   timecop (~> 0.7)
   twilio-ruby (~> 3.12)
-
-BUNDLED WITH
```
Comment (ericcf) [inline Gemfile.lock]: Are you using a different version of bundler?

Comment (michaelwehrley) [inline Gemfile.lock]: Not sure; I will give this a check.

Comment (michaelwehrley) [inline Gemfile.lock]: I was using 1.9.4.

File: app/models/task_status.rb [hunk 2]
```diff
@@ -85,31 +77,11 @@ def mark_complete
   end
 
   def is_lesson?
-    task.bit_core_content_module.type == "ContentModules::LessonModule"
+    task.bit_core_content_module.type == LESSON_MODULE_TYPE
```
Comment (ericcf) [inline app/models/task_status.rb]: Could you just use this here?
```ruby
task.bit_core_content_module.instance_of? ContentModules::LessonModule
```

**Comment by ericcf** [comment]
Other than my minor comment - looks good!

File: app/models/task_status.rb (hunk 2 above)
Comment (michaelwehrley) [inline app/models/task_status.rb]: More clear.  Thanks.

**Comment by michaelwehrley** [comment]
Merging.  Thanks!

**Pull request merged**